### PR TITLE
Changelog 3.2.0

### PR DESCRIPTION
**Pull request by bam241**
Merging the last news in the changelog

### CODE REVIEW

**Comment by bam241** [comment]
@pshriwise @gonuke this should be ready to look at !

File: doc/CHANGELOG.rst [hunk 1]
```diff
@@ -174,7 +229,10 @@ v3.2.0
 * Direct Travis to grab the docker image from the cnerg dockerhub account
   instead of Lucas's account. (PR#546)
 
-**Deprecated:** None
+**Deprecated:** 
+* DagMC: Deprecated constructor using a raw pointer for the MBI instance,
```
Comment (pshriwise) [inline doc/CHANGELOG.rst:255]: We'll need an empty line between the header and content for this to render correctly I think.

**Review by pshriwise — CHANGES_REQUESTED** [review]
Small format comment, but otherwise this looks good to me!

File: doc/CHANGELOG.rst (hunk 1 above)
Comment (bam241) [reply doc/CHANGELOG.rst:255]: done !

**Review by pshriwise — APPROVED** [review]

**Comment by bam241** [comment]
still having issue with the way I shaped the CHANGELOG for the docs...

still working on fixing this.

**Comment by bam241** [comment]
this is now fixed.
This is good to go on my side.

**Comment by pshriwise** [comment]
I'm getting viewer errors still for the second level of nesting in the list. It seems that the continuation lines aren't allowed in the second level of the bullets?

Also, an empty line between the levels seems necessary too.

So things like:

```
* Default to a Release build. This results in optimization flags being used                                                                                                                                                                                   
  everywhere as appropriate. (PR#555)                                                                                                                                                                                                                         
  * Note that MCNP is still configured to use no more than ``-O1``                                                                                                                                                                                            
* Add macros to ``cmake/DAGMC_macros.cmake``. This results in much less                                                                                                                                                                                       
  duplicated cmake elsewhere. The following macros were added: (PR#555)                                                                                                                                                                                       
  * ``dagmc_setup_build``: Sets core variables used throughout the rest of the                                                                                                                                                                                
  project.                                                                                                                                                                                                                                                    
  * ``dagmc_setup_options``: Defines cmake build options.                                                                                                                                                                                                     
  * ``dagmc_setup_flags``: Defines compiler flags.                          
```

Need to become: 
```
* Default to a Release build. This results in optimization flags being used                                                                                                                                                                                   
  everywhere as appropriate. (PR#555)                                                                                                                                                                                                                         
                                                                                                                                                                                                                                                              
  * Note that MCNP is still configured to use no more than ``-O1``                                                                                                                                                                                            
* Add macros to ``cmake/DAGMC_macros.cmake``. This results in much less                                                                                                                                                                                       
  duplicated cmake elsewhere. The following macros were added: (PR#555)                                                                                                                                                                                       
                                                                                                                                                                                                                                                              
  * ``dagmc_setup_build``: Sets core variables used throughout the rest of the project.                                                                                                                                                                       
  * ``dagmc_setup_options``: Defines cmake build options.                                                                                                                                                                                                     
  * ``dagmc_setup_flags``: Defines compiler flags. 
```

This is all based on using the `restview`, but sphinx may be smarter than that? Correct me if I'm wrong here @bam241.

**Comment by bam241** [comment]
I have no clue

File: doc/CHANGELOG.rst [hunk 2]
```diff
@@ -31,17 +36,19 @@ v3.2.0
   * Note that MCNP is still configured to use no more than ``-O1``
```
Comment (pshriwise) [inline doc/CHANGELOG.rst:37]:
```suggestion

  * Note that MCNP is still configured to use no more than ``-O1``
```

File: doc/CHANGELOG.rst [hunk 3]
```diff
@@ -51,15 +58,69 @@ v3.2.0
...
+* PullRequest-Agent suggestions DagMC (PR#671, PR#676):
+
+  * updated pointer management to RAII ("Resource Allocation Is Initialization") technique, MBI is now a shared_ptr unless passed 
+    as a raw pointer in the DagMC constructor (can be returned as a shared_ptr if not provided as a raw pointer),  GTT is now a 
+    shared_ptr, and can only be returned as such, GQT is now a uniq_ptr, (and can't be return - not change there)
+  * tests: DagMC instance is now a shared_ptr, when used, MBI instance is now a shared_ptr
+
+* PullRequest-Agent suggestions uwuw, tally, overlap_check, build_obb, misc/tests (PR#680)
+* PullRequest-Agent suggestions Geat4 (PR#691)
+* Updates to variable names in make_watergight files (PR#672)
```
Comment (pshriwise) [inline doc/CHANGELOG.rst]:
```suggestion
* Updates to variable names in make_watertight files (PR#672)
```

**Review by pshriwise — CHANGES_REQUESTED** [review]
One more format update and a typo correction.

**Comment by bam241** [comment]
@pshriwise done!

**Review by pshriwise — APPROVED** [review]
Sweet. This is looking good to me now. Thanks for compiling all of this @bam241!

**Pull request merged**